clang/include/clang/Driver/Options.td

```diff
@@ -2533,6 +2533,31 @@ def fno_sanitize_merge_handlers : Flag<["-"], "fno-sanitize-merge">, Group<f_cla
                         Alias<fno_sanitize_merge_handlers_EQ>, AliasArgs<["all"]>,
                         Visibility<[ClangOption, CLOption]>,
                         HelpText<"Do not allow compiler to merge handlers for any sanitizers">;
+def fsanitize_annotate_debug_info_EQ
+    : CommaJoined<["-"], "fsanitize-annotate-debug-info=">,
+      Group<f_clang_Group>,
+      HelpText<"Annotate checks with debug info for specified sanitizers, if "
+               "supported">;
```

Comment:
We can afford to be a bit more verbose in the help text. What about something like this?

```suggestion
      HelpText<"Annotate sanitizer instrumentation with extra debug info for the specified sanitizers, if "
               "supported">;
```

My thinking is

* Replace `checks` with `instrumentation` because not all instrumentation is necessarily a check. E.g. I think stack poisoning is done with instrumentation and that's not a "check".
* Say "extra debug info" since this is additional information on top of normal debug info
* Missing "the"